tools/archetype/binding/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,52 @@
+# <bindingName> Binding
+
+_Give some details about what this binding is meant for - a protocol, system, specific device._
+
+_If possible, provide some resources like pictures, a YouTube video, etc. to give an impression of what can be done with this binding._
+
+## Supported Things
+
+_Please describe the different supported things / devices within this section._
+_Which different types are supported, which models were tested etc.?_
+_Note that it is planned to generate some part of this based on the XML files within ```ESH-INF/thing``` of your binding._
+
+## Discovery
+
+_Describe the available auto-discovery features here. Mention for what it works and what needs to be kept in mind when using it._
+
+## Binding Configuration
+
+_If your binding requires or supports general configuration settings, please create a folder ```cfg``` and place the configuration file ```<bindingId>.cfg``` inside it. In this section, you should link to this file and provide some information about the options. The file could e.g. look like:_
```

Comment:
Actually, making it a separate folder than doc was by intention. My plan is to use this ```cfg``` folder as a source for the Karaf features, so that they can bring the required configuration with the bundles. Maybe we have to move the cfg files to some completely other place then, but we will see this when doing the Karaf port. But the point is: The cfg files are more than "just" documentation :-)